### PR TITLE
WIP YOLOv4 - model1

### PR DESCRIPTION
**Pull request by mariaogryczak**

### CODE REVIEW

File: yolov4/model1/README.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
+# Pytorch-YOLOv4
```
Comment (majsylw) [inline yolov4/model1/README.md:1]: Please consider shorten this description - leave only necessary things

File: yolov4/model1/Use_yolov4_to_train_your_own_data.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+yolov4的发布引起了不少的关注，但由于darknet是大佬c语言写的，对代码的阅读有诸多不变，所以周末的时候写了个pytorch版的(蹭一波热度)。虽然pytorch——yolov4写好已经有一段时间了，但是由于种种原因一直没有进行验证(主要就是懒)，大家提出了诸多问题帮助修复很多bug，还有大佬一起增加新的功能，感谢大家的帮助。这些天呼声最高的就是如何如何使用自己的数据进行训练，昨天又是周末，就把这件拖了很久的事做了。并不像使用很多数据，于是自己制作了一个简单的数据集。
```
Comment (majsylw) [inline yolov4/model1/Use_yolov4_to_train_your_own_data.md:1]: This file is not in english, please remove it

File: yolov4/model1/data/val.txt [hunk 3]
```diff
@@ -0,0 +1,215 @@
+not_annotated/48580364331_4189dcb929_z.jpg 140,370,187,199,6 332,197,487,463,6
```
Comment (majsylw) [inline yolov4/model1/data/val.txt:1]: we don't want to store annotations inside repository, please remove it

File: yolov4/model1/generateTrainTXT.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+import pylab
+
+import taco_to_dataset
+
+module_path = str(Path.cwd().parents[0] / "src")
+if module_path not in sys.path:
+    sys.path.append(module_path)
+
+
+dataset_path = '/dih4/dih4_2/wimlds/TACO-master/data'
```
Comment (majsylw) [inline yolov4/model1/generateTrainTXT.py:25]: this paths can be put in argparse to allow user put them in command line

File: yolov4/model1/taco_to_dataset.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# convert taco label to detect-waste labels
```
Comment (majsylw) [inline yolov4/model1/taco_to_dataset.py:1]: I think we have this script in common files in ```utils``` directory

File: yolov4/model1/train.txt [hunk 6]
```diff
@@ -0,0 +1,322 @@
+not_annotated/49056092946_b59e47c26d_z.jpg 301,246,369,368,0 183,298,218,207,6 317,280,357,393,0 129,247,215,209,0
```
Comment (majsylw) [inline yolov4/model1/train.txt:1]: Again - we don't want to store predictions in repository

**Comment by AgaMiko** [comment]
Hello @mariaogryczak
if your commit is ready to pull then please squash and merge.

**Pull request closed without merging**